### PR TITLE
Google Source connector licenses to Elv2

### PR DESCRIPTION
**Pull request by evantahler**
See https://github.com/airbytehq/airbyte/issues/27724

### CODE REVIEW

**Review by brianjlai — APPROVED** [review]

**Comment by octavia-squidington-iii** [comment]
##  source-google-sheets test report (commit [`d9b930cc69`](https://github.com/airbytehq/airbyte/pull/27738/commits/d9b930cc69d3310c382c27c1ed00e17a3e7145b1)) - ❌

⏲️  Total pipeline duration: 263 seconds

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-sheets/metadata.yaml | ✅        |
| Connector version semver check                                              | ✅        |
| Connector version increment check                                           | ❌        |
| QA checks                                                                   | ❌        |
| Code format checks                                                          | ✅        |
| Connector package install                                                   | ✅        |
| Build source-google-sheets docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                  | ✅        |
| Acceptance tests                                                            | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/evan_elv2-google/1687827549/d9b930cc69d3310c382c27c1ed00e17a3e7145b1/source-google-sheets/0.2.39/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-sheets test
```

File: airbyte-integrations/connectors/source-google-sheets/metadata.yaml [hunk 1]
```diff
@@ -9,11 +9,11 @@ data:
   dockerRepository: airbyte/source-google-sheets
   githubIssueLabel: source-google-sheets
   icon: google-sheets.svg
-  license: MIT
+  license: Elv2
   name: Google Sheets
   registries:
     cloud:
-      dockerImageTag: 0.2.21
+      dockerImageTag: 0.3.0
```
Comment (brianjlai) [inline airbyte-integrations/connectors/source-google-sheets/metadata.yaml:16]: as per failing tests, need to bump the dockerImageTag on line 8

**Comment by octavia-squidington-iii** [comment]
##  source-google-ads test report (commit [`d9b930cc69`](https://github.com/airbytehq/airbyte/pull/27738/commits/d9b930cc69d3310c382c27c1ed00e17a3e7145b1)) - ✅

⏲️  Total pipeline duration: 895 seconds

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-ads/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| Connector version increment check                                        | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-google-ads docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                               | ✅        |
| Acceptance tests                                                         | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/evan_elv2-google/1687827549/d9b930cc69d3310c382c27c1ed00e17a3e7145b1/source-google-ads/0.3.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-ads test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-search-console test report (commit [`d9b930cc69`](https://github.com/airbytehq/airbyte/pull/27738/commits/d9b930cc69d3310c382c27c1ed00e17a3e7145b1)) - ✅

⏲️  Total pipeline duration: 913 seconds

| Step                                                                                | Result   |
|:------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-search-console/metadata.yaml | ✅        |
| Connector version semver check                                                      | ✅        |
| Connector version increment check                                                   | ✅        |
| QA checks                                                                           | ✅        |
| Code format checks                                                                  | ✅        |
| Connector package install                                                           | ✅        |
| Build source-google-search-console docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                          | ✅        |
| Acceptance tests                                                                    | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/evan_elv2-google/1687827549/d9b930cc69d3310c382c27c1ed00e17a3e7145b1/source-google-search-console/1.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-search-console test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-analytics-v4 test report (commit [`d9b930cc69`](https://github.com/airbytehq/airbyte/pull/27738/commits/d9b930cc69d3310c382c27c1ed00e17a3e7145b1)) - ✅

⏲️  Total pipeline duration: 1374 seconds

| Step                                                                              | Result   |
|:----------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-analytics-v4/metadata.yaml | ✅        |
| Connector version semver check                                                    | ✅        |
| Connector version increment check                                                 | ✅        |
| QA checks                                                                         | ✅        |
| Code format checks                                                                | ✅        |
| Connector package install                                                         | ✅        |
| Build source-google-analytics-v4 docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                        | ✅        |
| Acceptance tests                                                                  | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/evan_elv2-google/1687827549/d9b930cc69d3310c382c27c1ed00e17a3e7145b1/source-google-analytics-v4/0.2.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-analytics-v4 test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-analytics-data-api test report (commit [`d9b930cc69`](https://github.com/airbytehq/airbyte/pull/27738/commits/d9b930cc69d3310c382c27c1ed00e17a3e7145b1)) - ✅

⏲️  Total pipeline duration: 1798 seconds

| Step                                                                                    | Result   |
|:----------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-analytics-data-api/metadata.yaml | ✅        |
| Connector version semver check                                                          | ✅        |
| Connector version increment check                                                       | ✅        |
| QA checks                                                                               | ✅        |
| Code format checks                                                                      | ✅        |
| Connector package install                                                               | ✅        |
| Build source-google-analytics-data-api docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                              | ✅        |
| Acceptance tests                                                                        | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/evan_elv2-google/1687827549/d9b930cc69d3310c382c27c1ed00e17a3e7145b1/source-google-analytics-data-api/1.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-analytics-data-api test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-sheets test report (commit [`1e7992ec86`](https://github.com/airbytehq/airbyte/pull/27738/commits/1e7992ec8608f459da3ff2c21010458123fa991f)) - ✅

⏲️  Total pipeline duration: 224 seconds

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-sheets/metadata.yaml | ✅        |
| Connector version semver check                                              | ✅        |
| Connector version increment check                                           | ✅        |
| QA checks                                                                   | ✅        |
| Code format checks                                                          | ✅        |
| Connector package install                                                   | ✅        |
| Build source-google-sheets docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                  | ✅        |
| Acceptance tests                                                            | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/evan_elv2-google/1687849663/1e7992ec8608f459da3ff2c21010458123fa991f/source-google-sheets/0.3.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-sheets test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-search-console test report (commit [`1e7992ec86`](https://github.com/airbytehq/airbyte/pull/27738/commits/1e7992ec8608f459da3ff2c21010458123fa991f)) - ✅

⏲️  Total pipeline duration: 813 seconds

| Step                                                                                | Result   |
|:------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-search-console/metadata.yaml | ✅        |
| Connector version semver check                                                      | ✅        |
| Connector version increment check                                                   | ✅        |
| QA checks                                                                           | ✅        |
| Code format checks                                                                  | ✅        |
| Connector package install                                                           | ✅        |
| Build source-google-search-console docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                          | ✅        |
| Acceptance tests                                                                    | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/evan_elv2-google/1687849663/1e7992ec8608f459da3ff2c21010458123fa991f/source-google-search-console/1.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-search-console test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-ads test report (commit [`1e7992ec86`](https://github.com/airbytehq/airbyte/pull/27738/commits/1e7992ec8608f459da3ff2c21010458123fa991f)) - ✅

⏲️  Total pipeline duration: 885 seconds

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-ads/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| Connector version increment check                                        | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-google-ads docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                               | ✅        |
| Acceptance tests                                                         | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/evan_elv2-google/1687849663/1e7992ec8608f459da3ff2c21010458123fa991f/source-google-ads/0.3.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-ads test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-analytics-v4 test report (commit [`1e7992ec86`](https://github.com/airbytehq/airbyte/pull/27738/commits/1e7992ec8608f459da3ff2c21010458123fa991f)) - ✅

⏲️  Total pipeline duration: 1286 seconds

| Step                                                                              | Result   |
|:----------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-analytics-v4/metadata.yaml | ✅        |
| Connector version semver check                                                    | ✅        |
| Connector version increment check                                                 | ✅        |
| QA checks                                                                         | ✅        |
| Code format checks                                                                | ✅        |
| Connector package install                                                         | ✅        |
| Build source-google-analytics-v4 docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                        | ✅        |
| Acceptance tests                                                                  | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/evan_elv2-google/1687849663/1e7992ec8608f459da3ff2c21010458123fa991f/source-google-analytics-v4/0.2.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-analytics-v4 test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-analytics-data-api test report (commit [`1e7992ec86`](https://github.com/airbytehq/airbyte/pull/27738/commits/1e7992ec8608f459da3ff2c21010458123fa991f)) - ✅

⏲️  Total pipeline duration: 1798 seconds

| Step                                                                                    | Result   |
|:----------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-analytics-data-api/metadata.yaml | ✅        |
| Connector version semver check                                                          | ✅        |
| Connector version increment check                                                       | ✅        |
| QA checks                                                                               | ✅        |
| Code format checks                                                                      | ✅        |
| Connector package install                                                               | ✅        |
| Build source-google-analytics-data-api docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                              | ✅        |
| Acceptance tests                                                                        | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/evan_elv2-google/1687849663/1e7992ec8608f459da3ff2c21010458123fa991f/source-google-analytics-data-api/1.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-analytics-data-api test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-sheets test report (commit [`93f37ed744`](https://github.com/airbytehq/airbyte/pull/27738/commits/93f37ed7444d3829b626a479337b820e0be8ef71)) - ❌

⏲️  Total pipeline duration: 210 seconds

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-sheets/metadata.yaml | ✅        |
| Connector version semver check                                              | ✅        |
| Connector version increment check                                           | ❌        |
| QA checks                                                                   | ✅        |
| Code format checks                                                          | ✅        |
| Connector package install                                                   | ✅        |
| Build source-google-sheets docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                  | ✅        |
| Acceptance tests                                                            | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/evan_elv2-google/1687933610/93f37ed7444d3829b626a479337b820e0be8ef71/source-google-sheets/0.3.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-sheets test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-ads test report (commit [`93f37ed744`](https://github.com/airbytehq/airbyte/pull/27738/commits/93f37ed7444d3829b626a479337b820e0be8ef71)) - ❌

⏲️  Total pipeline duration: 859 seconds

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-ads/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| Connector version increment check                                        | ❌        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-google-ads docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                               | ✅        |
| Acceptance tests                                                         | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/evan_elv2-google/1687933610/93f37ed7444d3829b626a479337b820e0be8ef71/source-google-ads/0.3.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-ads test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-search-console test report (commit [`93f37ed744`](https://github.com/airbytehq/airbyte/pull/27738/commits/93f37ed7444d3829b626a479337b820e0be8ef71)) - ❌

⏲️  Total pipeline duration: 909 seconds

| Step                                                                                | Result   |
|:------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-search-console/metadata.yaml | ✅        |
| Connector version semver check                                                      | ✅        |
| Connector version increment check                                                   | ❌        |
| QA checks                                                                           | ✅        |
| Code format checks                                                                  | ✅        |
| Connector package install                                                           | ✅        |
| Build source-google-search-console docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                          | ✅        |
| Acceptance tests                                                                    | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/evan_elv2-google/1687933610/93f37ed7444d3829b626a479337b820e0be8ef71/source-google-search-console/1.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-search-console test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-analytics-v4 test report (commit [`93f37ed744`](https://github.com/airbytehq/airbyte/pull/27738/commits/93f37ed7444d3829b626a479337b820e0be8ef71)) - ❌

⏲️  Total pipeline duration: 1124 seconds

| Step                                                                              | Result   |
|:----------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-analytics-v4/metadata.yaml | ✅        |
| Connector version semver check                                                    | ✅        |
| Connector version increment check                                                 | ❌        |
| QA checks                                                                         | ✅        |
| Code format checks                                                                | ✅        |
| Connector package install                                                         | ✅        |
| Build source-google-analytics-v4 docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                        | ✅        |
| Acceptance tests                                                                  | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/evan_elv2-google/1687933610/93f37ed7444d3829b626a479337b820e0be8ef71/source-google-analytics-v4/0.2.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-analytics-v4 test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-analytics-data-api test report (commit [`93f37ed744`](https://github.com/airbytehq/airbyte/pull/27738/commits/93f37ed7444d3829b626a479337b820e0be8ef71)) - ❌

⏲️  Total pipeline duration: 1754 seconds

| Step                                                                                    | Result   |
|:----------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-analytics-data-api/metadata.yaml | ✅        |
| Connector version semver check                                                          | ✅        |
| Connector version increment check                                                       | ❌        |
| QA checks                                                                               | ✅        |
| Code format checks                                                                      | ✅        |
| Connector package install                                                               | ✅        |
| Build source-google-analytics-data-api docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                              | ✅        |
| Acceptance tests                                                                        | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/evan_elv2-google/1687933610/93f37ed7444d3829b626a479337b820e0be8ef71/source-google-analytics-data-api/1.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-analytics-data-api test
```

**Pull request merged**